pkg/cacheutil/memcached_client.go

```diff
@@ -269,7 +269,7 @@ func (c *memcachedClient) Stop() {
 	c.workers.Wait()
 }
 
-func (c *memcachedClient) SetAsync(ctx context.Context, key string, value []byte, ttl time.Duration) error {
+func (c *memcachedClient) SetAsync(_ context.Context, key string, value []byte, ttl time.Duration) error {
```

Comment:
Seems like the changes in this file are unrelated? :eye:

Reply:
This was done by me in the PR #2775. I think this proposal branch has been created from that PR branch or something like this.